README.md

```diff
@@ -30,38 +34,70 @@ Working together as an Agile group, we worked hard to redesign this application
   - Jordan - refactor code editor, refactoring alert messages
   - Joe - Authentication-login routing, testing, minor fixes
   - Daniel - Interview space design
-
+___
+___
 ## Technologies Used
 
-- HTML5, CSS3, JavaScript, express.js, node.js,
-- React, react-router, react-auth,
-- Bootstrap, jest,
-- Docker, PostgreSQL, Git, NPM,
-- Visual Studio Code, Github Projects,
-- Discord, Slack, Zoom
+
+![Coding Languages](https://img.shields.io/github/languages/count/gschool-blue-ocean/admissions-hub-20?label=Coding%20Languages&style=plastic)
+
+ [![JavaScript](https://img.shields.io/badge/-JavaScript-F7DF1E?style=flat&logo=javascript&logoColor=black)](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+  [![HTML](https://img.shields.io/badge/-HTML-E34F26?style=flat&logo=html5&logoColor=black)](https://developer.mozilla.org/en-US/docs/Web/HTML)
+  [![CSS](https://img.shields.io/badge/-CSS-1572B6?style=flat&logo=css3&logoColor=white)](https://developer.mozilla.org/en-US/docs/Web/CSS)
+  [![React](https://img.shields.io/badge/-React-61DAFB?style=flat&logo=react&logoColor=black)](https://react.dev/)
+  [![React Router](https://img.shields.io/badge/-React_Router-red?style=flat&logo=reactrouter&logoColor=black)](https://reactrouter.com/en/main)
+  [![React-Auth](https://img.shields.io/badge/-React_Auth-red?style=flat&logo=npm&logoColor=black)](https://www.npmjs.com/package/react-auth-kit)
+  [![Jest](https://img.shields.io/badge/-Jest-purple?style=flat&logo=jest&logoColor=black)](https://jestjs.io/)
+  [![Bootstrap](https://img.shields.io/badge/-Bootstrap-7952B3?style=flat&logo=bootstrap&logoColor=white)](https://getbootstrap.com/)
+  [![Vite](https://img.shields.io/badge/-Vite-646CFF?style=flat&logo=vite&logoColor=F6DC40)](https://vitejs.dev/)
+  
+  [![Node.js](https://img.shields.io/badge/-Node.js-339933?style=flat&logo=Node.js&logoColor=black)](https://nodejs.org/)
+  [![Express.js](https://img.shields.io/badge/-Express.js-000000?style=flat&logo=express&logoColor=white)](https://expressjs.com/)
+  [![Supabase](https://img.shields.io/badge/-Supabase-3FCF8E?style=flat&logo=supabase&logoColor=black)](https://supabase.com/)
+  [![Code Sandbox](https://img.shields.io/badge/Code%20Sandbox-Live%20code%20editor-blue?style=plastic&logo=codesandbox)](https://codesandbox.io/?from-app=1)
+
```

Comment:
i think socket.io is a worth mention if that has a badge

Reply:
Ohh, thank you thats right I forgot thank you

Reply:
Done